### PR TITLE
Update logic for corner radius

### PR DESCRIPTION
**Pull request by kerrmarin**

### CODE REVIEW

File: lib/processLayerStyles.js [hunk 1]
```diff
@@ -37,10 +37,16 @@ const processLayerStyles = async (layerStyles, filepath, outputFolder, modifiers
         // Border is aplied after the cornerRadius because we need to know the radius for the border
         if (currentModifier.modifier != null) {
           const radius = cornerRadius(currentModifier.modifier)
-          addOptionalProperty("overlay", data.style, layerBorder(layerStyle, radius))
-
-          // Clip the shape for the corner radius
-          addOptionalProperty("clipShape", data.style, cornerRadius(currentModifier.modifier) )
+          // If there is a radius we overlay it and clip it
+          if (radius) {
```
Comment (atomoil) [inline lib/processLayerStyles.js:41]: this check would be great for swift, but for javascript there is all kinds of falsey / truthy shizzle to deal with. What does cornerRadius return if it doesn't have one? Undefined or null presumably - can the check specifically check for that and for radius == 0?

Comment (kerrmarin-lvmh) [reply lib/processLayerStyles.js:41]: Sure, I can check for `null` since radius is an object

File: lib/processLayerStyles.js [hunk 2]
```diff
@@ -37,10 +37,16 @@ const processLayerStyles = async (layerStyles, filepath, outputFolder, modifiers
...
-          // Clip the shape for the corner radius
-          addOptionalProperty("clipShape", data.style, cornerRadius(currentModifier.modifier) )
+          // If there is a radius we overlay it and clip it
+          if (radius === null) {
+            addOptionalProperty("overlay", data.style, layerBorder(layerStyle, radius))
+
+            // Clip the shape for the corner radius
+            addOptionalProperty("clipShape", data.style, cornerRadius(currentModifier.modifier) )
+          } else {
+            // If there is no radius or the radius is 0, then use a regular border
```
Comment (atomoil) [inline lib/processLayerStyles.js:47]: either this comment is wrong or your check is wrong...

Comment (kerrmarin-lvmh) [reply lib/processLayerStyles.js:47]: Ah sure, if the radius is `0` the `cornerRadius` function returns `null`

Comment (atomoil) [reply lib/processLayerStyles.js]: **BUENO**

**Review by atomoil — APPROVED** [review]

**Pull request merged**